changelogs/fragments/80-update_docker_connection_plugin.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
```

I'd rather consider this a bug fix than a feature:
```suggestion
bugfixes:
```